src/components/SearchResults.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "../styles/searchresults.css";
+import PropTypes from "prop-types"
+
+const SearchResults = ({ results }) => {
+
+    if (!results.length) {
+        return <p>No results</p>
+    } else {
+        return (
+            <div className="search-results">
+                <h1>Image Results:</h1>
+                {
+                    results.map((image, index) => (
+                        <img
+                            key={index}
+                            className="search-results__images"
+                            src={image}
+                            alt="search-results__image"
+                            data-testid="search-results-test"
+                        />
+                    ))
+                }
+            </div>
+        )
+    }
+}
+SearchResults.defaultProps = {
```

Comment:
If the props coming in are all required, there is no need for a defaultProps object. If it weren't required, `results` should be set to an array here so it matches the definition you have in the proptypes object